### PR TITLE
Axel

### PR DESCRIPTION
**Pull request by Axel2017**
This add the recording of hands,camera and audio and will get triggered once the alignement step is done.

### CODE REVIEW

File: Assets/Scripts/json_anchor.cs
Comment (MihneaToader) [inline Assets/Scripts/json_anchor.cs:1]: What is the purpose of this file? Looks to me like it's not used anywhere

File: Assets/Scripts/LogBoneLocation.cs [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    //         boneData["RotationX"] = worldRotation.x;
+    //         boneData["RotationY"] = worldRotation.y;
+    //         boneData["RotationZ"] = worldRotation.z;
+    //         boneData["RotationW"] = worldRotation.w;
+
+    //         numbersPerBone[$"{bone.Id}"] = boneData;
+    //     }
+
+    //     data.Entries.Add(new JsonEntry { Timestamp = timestamp, Position_rotation = numbersPerBone });
+    // }
```
Comment (MihneaToader) [inline Assets/Scripts/LogBoneLocation.cs]: Let's just get rid of this

File: Assets/Scripts/json_anchor.cs
Comment (Axel2017) [reply Assets/Scripts/json_anchor.cs:1]: This was for recording the anchors but we dont use it so can get rid of it.

File: Assets/Scripts/LogBoneLocation.cs (hunk 1 above)
Comment (Axel2017) [reply Assets/Scripts/LogBoneLocation.cs]: yes

**Pull request merged**